skins/GrowthDefi.scss

```
@@ -0,0 +1,10 @@
+.GrowthDefi {
```

The css selector should similar to the space folder and do not contain uppercases.
```suggestion
.growthdefi {
```

---

Changed to growthdefi.